earth_enterprise/BUILD.md

```diff
@@ -7,6 +7,8 @@ Building is currently supported on 64-bit versions of Ubuntu 14.04 LTS, Ubuntu 1
 
 ## GEE 5.2.1 Build Prerequisites (All platforms) 
 
+Setup may be performed manually (steps listed below), or by executing the [build script](../scripts/get-gee-build-environment.sh).
```

Comment:
You should probably indicate that this will install git-lfs, but it does not clone the repo for you.

Reply:
Or maybe move this sentence to within the setup step so it's not implied that it does anything else. We should also include a link to directly download the build script.

Reply:
This leads me to the elephant in the room: where can we put it such that it can be downloaded directly? The link there will only open it within github. Any thoughts?

Reply:
Moved sentence for clarity.